### DISCUSSION TITLE
CSS not being applied to pabloocal.github.io

### DISCUSSION THREAD

**Discussion by pabloocal** [community/community#22679] [Other Feature Feedback, Questions, & Ideas]
<p>Hello GitHub Community,</p>
<p>I am trying to transfer my website from Texas A&amp;M (math . tamu . edu /~pso/) to GitHub (pabloocal . github . io), but my css file is not being applied to my pages.</p>
<p>I have looked over similar issues from other users, including detailed solutions like the ones provided here: <a href="https://github.community/t/css-not-being-applied-in-pages/10466/10" class="inline-onebox">CSS not being applied in pages :/</a>, but I cannot find those errors in my documents.</p>
<p>Link: <a href="https://github.com/pabloocal/pabloocal.github.io" rel="nofollow noopener">https://github.com/pabloocal/pabloocal.github.io</a></p>
<p>Thank you very much for your help!</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/pabloocal">@pabloocal</a>, welcome to the GitHub Support Community!</p>
<p>I believe the issue is that your css folder begins with an underscore. When jekyll processing is enabled, files/folders that begin with underscores which aren’t essential will be ignored. Since it doesn’t look like you’re using jekyll, creating an empty file named <strong>.nojekyll</strong> in the root of your repository will disable jekyll processing and should resolve the issue.</p>

**Comment by pabloocal** [comment]
<p>Hello <a class="mention" href="/SethClydesdale">@SethClydesdale</a>, thank you very much for your reply!</p>
<p>I now have an empty file named .nojekyll in my root. Unfortunately, it seems that the issue persists, do you know if there may be other reasons why it’s not working?</p>
<p>I also tried renaming the _css file to just css (and changing the name in my home index.html file), but it also didn’t work.</p>
<p><strong>EDIT:</strong> I ignore why but after a few hours, it did work. Thank you again!</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="pabloocal" data-post="3" data-topic="129597">
<div class="title">
<div class="quote-controls"></div>
 pabloocal:</div>
<blockquote>
<p>I ignore why but after a few hours, it did work.</p>
</blockquote>
</aside>
<p>Changes to the sources of a GH Pages website are not propagated in real-time, so you might have to wait for a little while for them to show up — usually a few minutes, but depending on the GH workload and possibly on caching along the network, it could take up to an hour.</p>
<p>When posting similar questions, remember to provide links to both the live website and its source repository, so that the problem can be easier to dig into.</p>
<p>If the problem is solved, you should mark <a class="mention" href="/SethClydesdale">@SethClydesdale</a>’s reply as the solution.</p>
